v03_pipeline/bin/pipeline_worker.py

```diff
@@ -53,13 +54,16 @@ def main():
             run_id = datetime.datetime.now(datetime.timezone.utc).strftime(
                 '%Y%m%d-%H%M%S',
             )
+            loading_run_task_params = {
+                'project_guids': lpr.projects_to_run,
+                'project_remap_paths': project_remap_paths,
+                'project_pedigree_paths': project_pedigree_paths,
+                'run_id': run_id,
+                **task_kwargs,
+            }
             tasks = [
                 UpdateVariantAnnotationsTableWithNewSamplesTask(
-                    project_guids=lpr.projects_to_run,
-                    project_remap_paths=project_remap_paths,
-                    project_pedigree_paths=project_pedigree_paths,
-                    run_id=run_id,
-                    **task_kwargs,
+                    **loading_run_task_params,
                 ),
                 *[
                     WriteProjectFamilyTablesTask(
```

Comment:
there's an argument for making this a luigi task with dynamic dependencies (it would make the task parameter inheritance cleaner, and I could reuse the task inside of the `WriteSuccessFileTask` requires) but that feels silly since this can be accomplished by a simple loop

Reply:
👍 Do we even need `WriteProjectFamilyTablesTask` here if it's in the requires of the `WriteSuccessFileTask`?

Reply:
nope!!